src/views/Home.vue

```diff
@@ -1,18 +1,47 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <CampaignContainer v-bind:campaigns="campaigns"/>
   </div>
 </template>
 
 <script>
 // @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue';
+import CampaignContainer from '@/components/Container.vue';
 
 export default {
   name: 'Home',
   components: {
-    HelloWorld,
+    CampaignContainer
   },
-};
+  data() {
+    return {
```

Comment:
Would this be where we would make our calls to the blockchain and BE to fetch the list of campaigns?
It seems like for this it will be a call to the blockchain for the list of campaign addresses: `const campaigns = await factory.methods.getDeployedCampaigns().call();` and then we could send that list of campaign addresses to the BE to get the rest of the data about the campaigns that we need.

Reply:
Precisely, we'll have to find the equivalent to 'getInitalProps()' that's found in your og codebase, to retrieve data.